### PR TITLE
Add Video block to storybook/docusaurus based on existing Volto EEA add-on

### PR DESCRIPTION
**Pull request by tarantilis**
https://taskman.eionet.europa.eu/issues/250783

Storybook:
https://62713c57bcbdf4004a3d99b8-cglrlywvik.chromatic.com/?path=/story/components-video--default

### CODE REVIEW

**Comment by g-stamatis** [comment]
@eea-jenkins build storybook

**Comment by avoinea** [comment]
@tarantilis Please also add the latest `autoPlay` and `loop` controls:

https://github.com/eea/volto-nextcloud-video-block/blob/master/src/NextCloud/schema.js#L53-L60

File: src/ui/Video/Video.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Container } from 'semantic-ui-react';
+import NextCloudEmbed from '@eeacms/volto-nextcloud-video-block/NextCloud/players/NextCloudEmbed';
```
Comment (avoinea) [inline src/ui/Video/Video.stories.jsx:3]: @tarantilis I don't think this should be a hard dependency to `@eeacms/volto-nextcloud-video-block`

File: src/ui/Video/Video.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+  </Container>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  data: {
+    url: 'https://cmshare.eea.europa.eu/s/Xw7kH2TJGCibHJf/download',
+    autoPlay: false,
+    loop: false,
```
Comment (tarantilis) [inline src/ui/Video/Video.stories.jsx:26]: @avoinea I have them loop & autoplay here inside a data object, as I saw it here:

https://github.com/eea/volto-nextcloud-video-block/blob/eeb32ef3ae94325b94cf68cdc73e1e99671016f4/src/NextCloud/players/NextCloudEmbed.jsx#L27

let me check the hard dependancy and if I can move them outside the data object

**Review by ichim-david — APPROVED** [review]

**Review by ichim-david — APPROVED** [review]

**Pull request merged**